src/neo/SmartContract/ApplicationEngine.OpCodePrices.cs

```diff
@@ -71,8 +71,8 @@ partial class ApplicationEngine
             [OpCode.DEPTH] = 60,
             [OpCode.DROP] = 60,
             [OpCode.NIP] = 60,
-            [OpCode.XDROP] = 400,
-            [OpCode.CLEAR] = 400,
+            [OpCode.XDROP] = 4000,
+            [OpCode.CLEAR] = 2000,
```

Comment:
A little arbitrary, if we clean arrays or maps it could be good, but otherwise should be cheaper....

Reply:
Maybe a balance is needed between average case & worst case?

Reply:
Have reverted